src/main/java/com/cs56fitnessapp/models/DietFacts.java

```diff
@@ -6,7 +6,7 @@
  * Last Updated: 10/1/17
  */
 
-public interface DietFacts {
+public interface git DietFacts {
```

Comment:
DietFacts interface got changed - probably when you were typing git commands. Please remove these changes.